src/manage_sql_nvts.c

```diff
@@ -36,6 +36,8 @@
 #include <string.h>
 #include <unistd.h>
 
+#include <gvm/base/cvss.h> // for get_cvss_score_from_base_metrics
```

Comment:
```suggestion
#include <gvm/base/cvss.h>
```

I don't think this kind of annotation is worth the overhead it introduces.  None of the Manager code does this.